evil-surround.el

```diff
@@ -101,6 +101,9 @@ Each item is of the form (OPERATOR . OPERATION)."
 (defvar evil-surround-last-deleted-left ""
   "The previously deleted LEFT region.")
 
+(defvar evil-surround-current-pair nil
+  "")
```

Comment:
Does the byte compiler demand a docstring? If so, you may as well explain _why_ this exists, even if _what_ it is is obvious from the name.

Reply:
I don't think a docstring is required by the byte-compiler, but I added one anyway.